dashboard/src/components/ChartView/ChartHeader.tsx

```diff
@@ -1,30 +1,42 @@
 import * as React from "react";
 import { Link } from "react-router-dom";
 
+import { IChartVersion } from "shared/types";
 import ChartIcon from "../ChartIcon";
+import ChartDeployButton from "./ChartDeployButton";
+import "./ChartHeader.css";
 
 interface IChartHeaderProps {
-  appVersion?: string;
   id: string;
   icon?: string;
   repo: string;
   description: string;
+  version: IChartVersion;
+  namespace: string;
 }
 
 class ChartHeader extends React.Component<IChartHeaderProps> {
   public render() {
-    const { appVersion, id, icon, repo, description } = this.props;
+    const { id, icon, repo, description, version, namespace } = this.props;
+    const appVersion = version.attributes.app_version;
     return (
       <header>
-        <div className="ChartView__heading margin-normal">
-          <ChartIcon icon={icon} />
-          <div className="title margin-l-small">
-            <h1 className="margin-t-reset">{id}</h1>
-            <h5 className="subtitle margin-b-normal">
-              {appVersion && <span>{appVersion} - </span>}
-              <Link to={`/catalog/${repo}`}>{repo}</Link>
-            </h5>
-            <h5 className="subtitle margin-b-reset">{description}</h5>
+        <div className="ChartView__heading margin-normal row">
+          <div className="col-1 ChartHeader_img">
```

Comment:
since we use the ChartIcon component, I would use `ChartHeader__icon` as the class name.